src/main/java/com/github/cloudyrock/mongock/Mongock.java

```diff
@@ -110,6 +112,19 @@ void execute() throws MongockException {
 
   }
 
+  //this is to force sonar issue
+  Object ex;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)